src/ColoredTabs/Item.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Text, TouchableWithoutFeedback, StyleSheet } from 'react-native'
+
+export default class Item extends Component {
+  static propTypes = {
+    id: PropTypes.string,
+    active: PropTypes.bool,
```

Comment:
Do we really need this in item? Active tab define from component, as props "selected".